ui/webui/test/check-installation-language

```diff
@@ -21,26 +21,27 @@ import sys
 # import Cockpit's machinery for test VMs and its browser test API
 TEST_DIR = os.path.dirname(__file__)
 sys.path.append(os.path.join(TEST_DIR, "common"))
+sys.path.append(os.path.join(TEST_DIR, "helpers"))
 sys.path.append(os.path.join(os.path.dirname(TEST_DIR), "bots/machine"))
 
+from common_helpers import CommonHelpers
+from language_helpers import LanguageHelpers
 from testlib import MachineCase, nondestructive, test_main  # pylint: disable=import-error
 
 @nondestructive
-class TestLanguage(MachineCase):
+class TestLanguage(MachineCase, CommonHelpers, LanguageHelpers):
```

Comment:
Yes, the never ending story of customized test case classes :-) We went through this iteration, settled on composition over inheritance and migrated to `pytest` to have more options in the future. We are stuck with `unittest` in this case, so let's at least use the composition.

Reply:
Good point, I totally forget about that. It could result in nasty issues with the false test detection and similar other issues.